src/components/shared/Accordion.tsx

```diff
@@ -24,11 +24,13 @@ export function Accordion({ entries }: StyledAccordionProps) {
         <MUIAccordion key={i}>
           <AccordionSummary
             expandIcon={<ExpandMoreIcon />}
+            aria-label={`panel-content-${i}`}
```

Comment:
This should not be here. We added it just for testing, should be removed

Reply:
aria-label removed